docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
+```
+
+- `datasources` base class, as that core feature of sedna require identifying the features and labels from data input, we specify that the first parameter for train/evaluate of the ML framework
+
+```python
+from sedna.datasources import BaseDataSource
+
+
+train_data = BaseDataSource(data_type="train")
+train_data.x = []
+train_data.y = []
+for item in mnist_ds.create_dict_iterator():
+     train_data.x.append(item["image"].asnumpy())
+     train_data.y.append(item["label"].asnumpy())
+```
+
+-  `sedna.core` contain all edge-cloud features,  Please note that each feature has its own parameters.
+- **Hard Example Mining Algorithms** in IncrementalLearning named `hard_example_mining`  
+
+```python
+from sedna.core.incremental_learning import IncrementalLearning
+
+hard_example_mining = IncrementalLearning.get_hem_algorithm_from_config(
+    threshold_img=0.9
+)
+
+# initial an incremental instance
+incremental_instance = IncrementalLearning(
+    estimator=Estimator,
+    hard_example_mining=hem_dict
+)
+
+# Call the interface according to the job state
+
+# train.py
+incremental_instance.train(train_data=train_data, epochs=epochs,
+                          batch_size=batch_size,
+                          class_names=class_names,
+                          input_shape=input_shape,
+                          obj_threshold=obj_threshold,
+                          nms_threshold=nms_threshold)
+
+# inference
+results, _, is_hard_example = incremental_instance.inference(
+            data, input_shape=input_shape)
+            
+
+```
+
+
+### 3. Configuration
+
+##### 3.1 Prepare Image
+This example uses the image:  
+```
+kubeedge/sedna-example-incremental-learning-helmet-detection:v0.4.0
+```
+
+This image is generated by the script [build_images.sh](/examples/build_image.sh), used for creating training, eval and inference worker.  
+
+##### 3.2 Create Incremental Job
+In this example, `$WORKER_NODE` is a custom node, you can fill it which you actually run.  
+
+```
+WORKER_NODE="edge-node" 
+```
+
+- Create Dataset  
+
+```
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Dataset
+metadata:
+  name: incremental-dataset
+spec:
+  url: "/data/helmet_detection/train_data/train_data.txt"
+  format: "txt"
+  nodeName: $WORKER_NODE
+EOF
+```
+
+- Create Initial Model to simulate the initial model in incremental learning scenario.  
+
+```
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Model
+metadata:
+  name: initial-model
+spec:
+  url : "/models/base_model"
+  format: "ckpt"
+EOF
+```
+
+- Create Deploy Model  
+
+```
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Model
+metadata:
+  name: deploy-model
+spec:
+  url : "/models/deploy_model/saved_model.pb"
+  format: "pb"
+EOF
+```
+
+
+### 4. Run
+
+* incremental learning supports hot model updates and cold model updates. Job support 
+cold model updates default. If you want to use hot model updates, please to add the following fields:  
```

Comment:
Job support cold model updates default.
->
Job support cold model updates by default.

Reply:
Copy from `examples`， shall we need to modify on both sides